rpc/src/dialect/check_tx.rs

```diff
@@ -78,9 +78,9 @@ where
             gas_used: msg.gas_used,
             events: msg.events.into_iter().map(Into::into).collect(),
             codespace: msg.codespace,
-            sender: msg.sender,
-            priority: msg.priority,
-            mempool_error: msg.mempool_error,
+            // sender: msg.sender,
+            // priority: msg.priority,
+            // mempool_error: msg.mempool_error,
```

Comment:
Should probably be restored based on my prior comment re: the priority mempool.

Reply:
https://github.com/informalsystems/tendermint-rs/pull/1312/commits/1f5688ddd542c7bfcd46b34b10c22eec0f7f2e0f